sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SonarDiagnosticAnalyzer.cs

```diff
@@ -30,5 +30,7 @@ public sealed override void Initialize(AnalysisContext context)
         }
 
         protected abstract void Initialize(SonarAnalysisContext context);
+
+        internal void InternalInitialize(SonarAnalysisContext context) => Initialize(context);
```

Comment:
I'm not sure I understand the purpose of this method. I think this is a code smell since it breaks encapsulation and if we really need the initialize to be internal we can just modify that, no?

Reply:
indeed.
I'll think of how to avoid the break of encapsulation